src/app/store/GlobalStore.tsx

```diff
@@ -144,6 +146,36 @@ export function unsafe_getFeatureFlag(key: FeatureName): boolean {
   return features[key].readyForRelease
 }
 
+/**
+ * This is marked as unsafe because it will not cause a re-render
+ * if used in a react component. Use `useExperimentVariant` instead.
+ * It is safe to use in contexts that don't require reactivity.
+ */
+export function unsafe_getExperiment(key: EXPERIMENT_NAME): IVariant | null {
+  const state = globalStoreInstance().getState() ?? null
+
+  if (state) {
+    const hasLocalOverride = !!state.artsyPrefs.experiments.localVariantOverrides[key];
+    const effectiveVariant = {
+      name:
+        state.artsyPrefs.experiments.localVariantOverrides[key] ||
+        state.artsyPrefs.experiments.unleashVariants[key]?.name ||
+        null,
+      enabled: hasLocalOverride
+        ? true
+        : state.artsyPrefs.experiments.unleashVariants[key]?.enabled || undefined,
+      payload: hasLocalOverride
+        ? (state.artsyPrefs.experiments.localPayloadOverrides?.[key] ?? undefined)
+        : state.artsyPrefs.experiments.unleashVariants[key]?.payload || undefined,
+    } as IVariant
+    return effectiveVariant
+  }
```

Comment:
I think we can improve the readability here with(easier to see what we return in case of local overried or not):
```suggestion
  if (!state) {
    if (__DEV__) {
      throw new Error(`Unable to access ${key} before GlobalStore bootstraps`)
    }
    return null
  }
  
  if (!state.artsyPrefs.experiments.localVariantOverrides[key]) {
    return {
      name: state.artsyPrefs.experiments.unleashVariants[key]?.name || null,
      enabled: state.artsyPrefs.experiments.unleashVariants[key]?.enabled || undefined,
      payload: state.artsyPrefs.experiments.unleashVariants[key]?.payload || undefined,
    }
  }
  
  return {
    name: ,
    enabled: true,
    payload: state.artsyPrefs.experiments.localPayloadOverrides?.[key] ?? undefined
  }
```